internal/codespaces/ssh.go

```diff
@@ -95,9 +83,56 @@ func newSSHCommand(ctx context.Context, port int, dst string, cmdArgs []string)
 	return cmd, connArgs, nil
 }
 
-// parseSSHArgs parses SSH arguments into two distinct slices of flags and command.
-// It returns an error if a unary flag is provided without an argument.
 func parseSSHArgs(args []string) (cmdArgs, command []string, err error) {
+	return parseArgs(args, "bcDeFIiLlmOopRSWw")
```

Comment:
We should update the usage help docs for the cp command. The ssh command shows this:

```
USAGE
  gh codespace ssh [<flags>...] [-- <ssh-flags>...] [<command>]
```

To let users know that `--` can be used to pass native SSH flags, current cp shows:

```
USAGE
  gh codespace cp [-e] [-r] <sources>... <dest>
```

We should update it to reflect this new capability.

Reply:
Updated and added an example